ufo/user_test.py

```diff
@@ -70,6 +90,167 @@ def testListUsersHandler(self, mock_get_all):
       details_link = flask.url_for('user_details', user_id=mock_users[x].id)
       self.assertEquals(details_link in user_list_output, True)
 
+  @patch.object(user, '_RenderUserAdd')
+  def testAddUsersGetHandler(self, mock_render):
+    """Test the add users get handler returns _RenderUserAdd's result."""
+    return_text = '<html>something here </html>'
+    mock_render.return_value = return_text
+    resp = self.client.get(flask.url_for('add_user'))
+
+    self.assertEquals(resp.data, return_text)
+
+  @patch('flask.render_template')
+  @patch.object(oauth, 'getSavedCredentials')
+  def testAddUsersGetNoCredentials(self, mock_get_saved_credentials,
+                                  mock_render_template):
+    """Test add user get should display an error when oauth isn't set."""
+    mock_get_saved_credentials.return_value = None
+    mock_render_template.return_value = ''
+
+    response = self.client.get(flask.url_for('add_user'))
+
+    args, kwargs = mock_render_template.call_args
+    self.assertEquals('add_user.html', args[0])
+    self.assertEquals([], kwargs['directory_users'])
+    self.assertEquals(kwargs['error'] is not None, True)
+
+  @patch('flask.render_template')
+  @patch.object(oauth, 'getSavedCredentials')
+  @patch.object(gds.GoogleDirectoryService, '__init__')
+  def testAddUsersGetNoParam(self, mock_ds, mock_get_saved_credentials,
+                            mock_render_template):
+    """Test add user get should display no users on initial get."""
+    mock_get_saved_credentials.return_value = FAKE_CREDENTIALS
```

Comment:
nit: plural?  Because when it's plural, I would expect a list.  But it's defined as a singleton string.